governance.md

```diff
@@ -0,0 +1,348 @@
+given a grace period of one year. If they don’t return to active participation
+within that time period they will be removed by vote of the Council without
+further grace period. All former Council members can be considered for
+membership again at any time in the future, like any other Project Contributor.
+Retired Council members will be listed on the project website, acknowledging
+the period during which they were active in the Council.
+An inactive BDFL may also be ejected by a consensus of current members under
+these same conditions.
+
+The Council reserves the right to eject current Members, other than the BDFL,
+if they are deemed to be actively harmful to the project’s well-being, and
+attempts at communication and conflict resolution have failed.
+
+
+### Conflict of interest
+
+It is expected that the Council Members and BDFL will be employed at a wide
+range of companies, universities and non-profit organizations. Because of this,
+it is possible that Members will have conflict of interests. Such conflict of
+interests include, but are not limited to:
+
+-   Financial interests, such as investments, employment, or contracting work,
+    outside of The Project that may influence their work on The Project.
+-   Access to proprietary information of their employer that could potentially
+    leak into their work with the Project.
+-   An issue where the person privately gains an advantage from The Project
+    resources, but The Project has no gain or suffers a disadvantage.
+
+All members of the Council, BDFL included, shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused his/herself for a particular decision, they will appoint a substitute
+BDFL for that decision.
+
+### Private communications of the Council
+
+Unless specifically required, all Council discussions and activities will be
+public and done in collaboration and discussion with the Project Contributors
+and Community. The Council will have a private mailing list that will be used
+sparingly and only when a specific matter requires privacy. When private
+communications and decisions are needed, the Council will do its best to
+summarize those to the Community after eliding personal/private/sensitive
+information that should not be posted to the public internet.
+
+### Subcommittees
+
+The Council can create subcommittees that provide leadership and guidance for
+specific aspects of the project. Like the Council as a whole, subcommittees
+should conduct their business in an open and public manner unless privacy is
+specifically called for. Private subcommittee communications should happen on
+the main private mailing list of the Council unless specifically called for.
+
+Question: if the BDFL is not on a subcommittee, do they still have override
+authority?
```

Comment:
This isn't clear to me, is it supposed to be part of the final document or expected to be discussed and properly formulated as a statement (i.e., without a question/suggestion)?

Reply:
Yes, I felt similarly when reading it.  It appears in the original document from Jupyter so I left it in.  I'd be happy to turn it into a statement though.

Reply:
IMHO, it makes more sense for this to be a statement. For people reading this, it may be very confusing to understand who's the question for.

Reply:
Done

Reply:
LGTM. +1